lib/workers/repository/dependency-dashboard.ts

```diff
@@ -263,6 +285,22 @@ export async function ensureDependencyDashboard(
     return;
   }
   let issueBody = '';
+
+  if (configMigrationRes.result === 'pr-exists') {
+    issueBody +=
+      '## Config Migration Needed\n\n' +
+      `See Config Migration PR: #${configMigrationRes.prNumber}.\n\n`;
+  } else if (configMigrationRes?.result === 'pr-modified') {
+    issueBody +=
+      '## Config Migration Needed\n\n' +
+      `The Config Migration branch exists but has been modified by another user. Renovate will not push to this branch unless it is first deleted. \n\n See Config Migration PR: #${configMigrationRes.prNumber}.\n\n`;
```

Comment:
How about something like this:

## Config Migration Needed (h2)

### Renovate will not update the Config Migration branch (h3)

Another user, or other bot, has pushed commits to the Config Migration branch. To be safe, Renovate will _not_ push new changes to the branch.

If you want Renovate to update the Config Migration branch: delete the branch, and let Renovate create a new branch.

> [!CAUTION]
> Deleting the Config Migration branch means losing the commits on the branch!

Reply:
Seems too much IMO. @rarkins WDYT??

Reply:
Changes:

- Drop caution (deleting branches is _obviously_ destructive, so we can probably rely on the user's common sense)
- Only cover the essential information

### Renovate will not update the config migration branch (h3)

Another user pushed commits to the Config Migration branch. Renovate will _not_ push new changes.

If you want Renovate to update the branch: delete the branch, and let Renovate create a new branch.

Reply:
We already have a header can we drop the h3 ?

Reply:
Yes

Reply:
> We already have a header can we drop the h3 ?

My point is that the heading: `## Config Migration Needed` only works for the happy path.

When Renovate shows the error message, I'd like to either:

- Swap `h2` heading for a heading like: `## Config Migration needed (error)`
- Or add a `h3` with a suitable error summary

Reply:
> Swap h2 heading for a heading like: ## Config Migration needed (error)

Going with this 🙌